tests/Umbraco.Tests.AcceptanceTest/cypress/typings.d.ts

```diff
@@ -3,3 +3,28 @@
 
 // type definitions for custom commands like "createDefaultTodos"
 // <reference types="support" />
+
+export {};
+
+declare global {
+    // eslint-disable-next-line @typescript-eslint/no-namespace
+    namespace Cypress {
+        interface Chainable<Subject> {
+            /**
+             * Checks to see if the language with specified name does not exist
+             * If it does it will automatically delete it
+             * @param  {string} name Name of language to delete
+             * @example cy.umbracoEnsureLanguageNotExists('Danish');
+             */
+            umbracoEnsureLanguageNotExists: (name: string) => Chainable<void>;
```

Comment:
As mentioned in your other PR we already have a method called cy.umbracoEnsureLanguageNameNotExists(name)
```suggestion
    namespace Cypress {
        interface Chainable<Subject> {
```